ansible/roles/jenkins-worker/tasks/main.yml

```diff
@@ -228,7 +228,7 @@
 
 - name: enable jenkins at startup - general
   when: not os|startswith("zos") and not os|startswith("macos")
-  service: name=jenkins state=started enabled=yes
+  service: name=jenkins state=restarted enabled=yes
```

Comment:
Does this work when the agent is not running?

Reply:
Just double checked on a machine, and looks to run ok